Missing java.sql dependency in MyFaces module on WildFly 39 (Java 21) causes NoClassDefFoundError: java/sql/ResultSet

When installing MyFaces 4.0.3 via the wildfly-myfaces-feature-pack 2.0.2 on WildFly 39.0.1.Final running on Java 21, the generated module definition for jakarta.faces.impl:myfaces does not include a dependency on the Java SE module java.sql.

MyFaces’ UIRepeat class attempts to load java.sql.ResultSet when creating a DataModel. I believe that on WildFly 39 with a Java 21 runtime image, Java SE modules are no longer implicitly visible to application modules unless explicitly declared. 

As a result, MyFaces fails with:

`Caused by: java.lang.NoClassDefFoundError: java/sql/ResultSet
    at org.apache.myfaces.view.facelets.component.UIRepeat.createDataModel(UIRepeat.java:251)`


### Environment:

WildFly 39.0.1.Final

Java 21.0.4

wildfly-myfaces-feature-pack 2.0.2

MyFaces 4.0.3

--- a/galleon-content/src/main/resources/modules/jakarta/faces/impl/myfaces/module.xml
+++ b/galleon-content/src/main/resources/modules/jakarta/faces/impl/myfaces/module.xml
@@ -30,6 +30,7 @@
         <module name="java.desktop"/>
         <module name="java.logging"/>
         <module name="java.naming"/>
+        <module name="java.sql"/>
         <module name="java.xml"/>
 
         <module name="org.jboss.logging"/>

